Makefile

```diff
@@ -2,18 +2,22 @@ NAME := tekton-pipeline
 CHART_DIR := charts/${NAME}
 
 CHART_REPO := gs://jenkinsxio/charts
+CHART_VERSION := 0.29.0
```

Comment:
How about setting the default as latest and then the user can do `make CHART_VERSION=0.29.0 fetch`?
something like `CHART_VERSION ?= latest` should work imo.